src/Dofs/ConstraintHandler.jl

```diff
@@ -54,13 +54,14 @@ struct ConstraintHandler{DH<:AbstractDofHandler,T}
     free_dofs::Vector{Int}
     values::Vector{T}
     dofmapping::Dict{Int,Int} # global dof -> index into dofs and values
+    bcvalues::Vector{BCValues{T}}
```

Comment:
So each new bc will have it's own bcvalues? Isn't it more a property of the field/interpolation?

Reply:
MixedDofHandler stores bcvalues in the fieldHandler, and DofHandler stores bcvalues itself, so in able to be able to have the same functionality for both Dohandlers, I must save the bcvalues in ConstraintHandler (I think)

Reply:
Maybe it is better to store it in `Dirichlet`